packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,550 @@
+>**Note:** The examples in this guide use Swift 5, SwiftUI, and a target of iOS 15 or later.
+
+The examples in this guide manage the sign-on flow using a singleton class (`SignInController`). The class handles asynchronous responses from the SDK by implementing the delegate functions of the `IDXClientDelegate` protocol. The SDK also provides completion handlers and Swift Concurrency (`async/await`).
```

Comment:
```suggestion
The examples in this guide manage the sign-in flow using a singleton class (`SignInController`). The class handles asynchronous responses from the SDK by implementing the delegate functions of the `IDXClientDelegate` protocol. The SDK also provides completion handlers and Swift Concurrency (`async/await`).
```

Reply:
Changed all occurrences of sign-on to sign-in. Will be in the next commit